client/tests/addons/inviting-an-editor-test.js

```diff
@@ -0,0 +1,54 @@
+import Ember from "ember";
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.